### DISCUSSION TITLE
Mac Version

### DISCUSSION THREAD

**Discussion by JohnEaglez** [TheFutureGadgetsLab/WaifuXL#160] [Ideas]
Any plans for a paid Mac app that runs on the Mac's neural engine, like the M1 Pro

**Comment by haydn-jones** [comment]
Yeah I can look into this. AFAIK PyTorch does support the neural engine to some extent. I'll look at ONNX's support as well. We're working on a command line tool in #157, so I imagine a lot of this functionality will be shared with that.

It will be free tho
